src/sass/shared/components/header.scss

```diff
@@ -156,12 +156,14 @@
   a {
     @include link-dark-bg;
     text-decoration: none;
+    padding: $padding-secondary-nav-item;
+    display: inline-block;
   }
 
   ul {
     li {
       display: inline-block;
-      padding: $padding-secondary-nav-item;
+      padding: 0;
```

Comment:
I think that `$padding-secondary-nav-item` var is no longer used anywhere, so I should remove it from the vars file
(That's bec I recommended to put there only relevant vars instead of every single dimmension)